frontends/web/src/routes/account/manage/manage.tsx

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, h, RenderableProps } from 'preact';
+import { route } from 'preact-router';
+import * as accountApi from '../../../api/account';
+import { getSupportedCoins } from '../../../api/backend';
+import SimpleMarkup from '../../../utils/simplemarkup';
+import { Button, Input } from '../../../components/forms';
+import { Entry } from '../../../components/guide/entry';
+import { Guide } from '../../../components/guide/guide';
+import { Header } from '../../../components/layout';
+import { translate, TranslateProps } from '../../../decorators/translate';
+import { Step, Steps } from './components/steps';
+import { CoinDropDown } from './components/coin-dropdown';
+import * as styles from '../manage/manage.css';
+import checkicon from '../../../assets/icons/check.svg';
+
+interface AddAccountProps {
+    // type?: 'multi' | 'btconly'
+}
+
+type Props = AddAccountProps & TranslateProps;
+
+interface State {
+    accountName: string;
+    coinCode: 'choose' | accountApi.CoinCode;
+    onlyOneSupportedCoin: boolean;
+    step: 0 | 1 | 2;
+    supportedCoins: string[];
+}
+
+class ManageAccount extends Component<Props, State> {
+    public readonly state: State = {
+        accountName: '',
+        coinCode: 'choose',
+        onlyOneSupportedCoin: false,
```

Comment:
This is strictly derived from `supportedCoins` below, so it's better to not have it in the state, but compute it on the fly.